core/conanfile.py

```diff
@@ -19,8 +22,12 @@ class EmbeddedPythonCore(ConanFile):
     options = {
         "version": ["ANY"],
         "openssl_variant": ["lowercase", "uppercase"],  # see explanation in `build_requirements()`
+        "zip_stdlib": ["no", "stored", "deflated"],
+    }
+    default_options = {
+        "openssl_variant": "lowercase",
+        "zip_stdlib": "stored",
```

Comment:
Is there a reason to not default to deflate?

Reply:
`deflated` comes with the expected tradeoff: smaller size but slower import time due to decompression overhead. The default in Python for zip imports is `stored` (see [`PyZipFile`](https://docs.python.org/3/library/zipfile.html#pyzipfile-objects)). I didn't have enough time to evaluate the performance hit on the import time, but I could compare the size difference and it's ~5% for the entire package (the standard library is only part of it). My conclusion from this was that it's not worth worrying about this for now. There's not much to gain right now, and we can re-evaluate down the line.

The nice thing is that we already get a nice size reduction just from `stored`. The `zip` only contains `.pyc` files and not `.py` so that already cuts the file count in half, and then the advantage of storing them as one file instead of many individual ones.